### ISSUE TITLE
Could not resolve 'wrapping_shift_left' in path

### ISSUE THREAD

**Issue by manylov** [zkworks-xyz/noir-base64#11]
nargo test generates errors like

```
error: Could not resolve 'wrapping_shift_left' in path
   ┌─ /Users/../nargo/github.com/zkworks-xyz/noir-base64v0.1.1/src/lib.nr:55:33
   │
55 │         let input1_last4 = std::wrapping_shift_left(input[1], 4) >> 2;
   │                                 -------------------
   │
```

**Comment by akonior** [comment]
Hi, thanks for submitting the issue. It is related to changes in the newest noir release. We will take care of this issue and hopefully fix it tomorrow.

**Comment by akonior** [comment]
Fixed in [v0.1.2](https://github.com/zkworks-xyz/noir-base64/releases/tag/v0.1.2) release
